src/redist/targets/BundledManifests.targets

```diff
@@ -0,0 +1,64 @@
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <ItemGroup>
+    <BundledManifests Include="Microsoft.NET.Workload.Android" Version="$(XamarinAndroidWorkloadManifestVersion)" WorkloadId="Microsoft.NET.Workload.Android" />
+    <BundledManifests Include="Microsoft.NET.Sdk.BlazorWebAssembly.AOT" Version="$(BlazorWorkloadManifestVersion)" WorkloadId="Microsoft.NET.Workload.BlazorWebAssembly" />
```

Comment:
I tried adding iOS/macOS packages too but was running into `error NU1213: The package Microsoft.NET.Workload.iOS 14.4.100-ci.pr.gh10810.1184 has a package type DotnetPlatform that is incompatible with this project.` @jonathanpeppers is this expected?

Reply:
I'm surprised the Android one worked. All of these packages have the `$(PackageType)` of `DotnetPlatform`, should they be something else?

I have only been able to download these packages using `@(PackageDownload)` directly.

Reply:
I'm not sure, @dsplaisted does it matter what the package type is for manifest packages?

Reply:
They should all be `DotnetPlatform`, and we should be using `PackageDownload`.

Reply:
FYI @pranavkm, looks like you'll want to change your package type.